style.css

```diff
@@ -1,8 +1,253 @@
+  width: 100%;
+  background-color: black;
+}
+
+.left-nav{
+  width: 70%;
+  align-items: center;
+  justify-content: flex-start;
+  position: sticky;
+}
+
+
+.logo img{
+  width: 140px;
+  padding: 3%;
+  margin-top: 1%;
+  cursor: pointer;
+}
+
+.navbar{
+  width: 85%;
+}
+
+.navbar a:hover {
+  text-decoration: underline;
+  text-decoration-color: red;
+}
+
+ul li{
+  color: #fff;
+  list-style: none;
+  font-size: 1rem;
+  margin-right: 40px;
+  cursor: pointer;
+}
+
+.logo{
+  width: 15%;
+}
+
+.search{
+  background-color: transparent;
+  border: 2px solid rgb(218, 20, 20);
+  border-radius: 8px;
+  color: #fff;
+  padding: 5px;
+  margin: 10px;
+  font-size: 1rem;
+  font-family: inherit;
+}
+
+.search:focus{
+  outline: 0;
+  background-color: transparent;
+}
+
+.search::placeholder{
+  color: #fff;
+}
+
+main{
+  display:flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 30px;
+}
+
+.movie {
+  width: 300px;
+  margin: 10px;
+  border-radius: 3px;
+  box-shadow: 0.2px 4px 5px rgba(0,0,0,0.1);
+  background-color: #fff;
+  position:relative;
+  overflow: hidden;
+}
+
+
+.movie img{
+  width:100%;
+}
+
+.movie-info{
+  color: black;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.5rem 1rem 1rem;
+  letter-spacing: 0.5px;
+}
+
+.movie-info h3{
+  margin-top: 0;
+}
+
+.movie-info span{
+  background-color: var(--primary-color);
+  padding: 0.25rem 0.5rem;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.movie-info span.green{
+  color:lightgreen;
+  background-color: black;
+}
+
+.movie-info span.orange{
+  color:orange;
+  background-color: black;
+}
+
+.movie-info span.red{
+  color:red;
+  background-color: black;
+}
+
+.overview{
+  position:absolute;
+  left:0;
+  right:0;
+  bottom:0;
+  background-color: #fff;
+  padding: 1rem;
+  max-height: 100%;
+  transform:translateY(101%);
+  transition:transform 0.3s ease-in;
+}
+
+.movie:hover .overview{
+  transform:translateY(0)
+}
+
+
+
+.togglebtn{
+  display: none;
+  font-size: 30px;
+  background-color: transparent;
+  border: none;
+  outline: none;
+  color: #0094da;
+ 
+}
+
+/* Media Queries*/
+
+@media only screen and (max-width: 1150px){
+
+  .title{
+    width: 400px;
+  }
+
+  .sub-title{
+    width: 280pt;
+    justify-content: flex-start;
+    padding: 0;
+    margin: 15px 0 0 10px; 
+
+  }
+  .banner h1{
+    font-size: 3.0rem;
+  }
+
+  .navbar{
+    width: 100%;
+    display: none;
+  }
+
+  .left-nav,
+  .right-nav{
+    width: 100%;
+    justify-content: flex-start;
+    margin: 20px 0;
+  }
+
+  ul li{
+    margin-right: 30px;
+  }
+
+  .togglebtn{
+    display: block;
+    font-size: 2rem;
+    color: #fff;
+    margin-top: 5px;
+    position: absolute;
+    right: 20px;
+    cursor: pointer;
+  }
+
+} 
+
+@media only screen and (max-width: 768px){
+ 
+  .banner h1{
+    font-size: 3rem;
+  }
+
+  .banner{
+    padding: 50px;
+  }
+
+
+}
+@media only screen and (max-width:550px){
+  .banner h1{
```

Comment:
The banner h1 receives the same font size from the media query, it is not needed to add it here.